app/lib/features/events/pages/event_details_page.dart

```diff
@@ -251,11 +255,18 @@ class _EventDetailPageConsumerState extends ConsumerState<EventDetailPage> {
             child: Column(
               crossAxisAlignment: CrossAxisAlignment.start,
               children: [
-                Text(
-                  calendarEvent.title(),
-                  maxLines: 2,
-                  overflow: TextOverflow.ellipsis,
-                  style: Theme.of(context).textTheme.titleMedium,
+                GestureDetector(
+                  onTap: () {
+                    if (canPostEvent) {
+                      showEditEventTitleBottomSheet(calendarEvent.title());
+                    }
+                  },
```

Comment:
considering that we also want longPress -> copy (next), shouldn't we use `InkWell` for these?

Reply:
Probably it will create any issue. 
Let share one demo with below screen recording for reference. 


https://github.com/acterglobal/a3/assets/51526480/c239748f-e14d-4aa5-aa35-6c05d84d4592